packages/next/pages/_error.tsx

```diff
@@ -20,7 +20,7 @@ export type ErrorProps = {
 export default class Error<P = {}> extends React.Component<P & ErrorProps> {
   static displayName = 'ErrorPage'
 
-  static getInitialProps({ res, err }: NextPageContext) {
+  static async getInitialProps({ res, err }: NextPageContext) {
```

Comment:
Not sure why `async` is required here given https://github.com/zeit/next.js/pull/7763

https://github.com/zeit/next.js/blob/be3eab2ad7892ca9a5be38f133494e700875804a/packages/next-server/lib/utils.ts#L16